apps/src/lib/tools/jsinterpreter/codegen.js

```diff
@@ -48,7 +48,7 @@ exports.strip = function (code) {
     // Strip out serial numbers.
     .replace(/(,\s*)?'block_id_\d+'\)/g, ')')
     // Remove timeouts.
-    .replace(INFINITE_LOOP_TRAP, '')
+    .replace(new RegExp(utils.escapeRegExp(INFINITE_LOOP_TRAP), 'g'), '')
```

Comment:
Coming soon to babel, `replaceAll` for this exact use case! https://twitter.com/v8js/status/1193917549060280320